### PR TITLE
Fix trailing color in new lines

### PR DESCRIPTION
**Pull request by eladchen**
When the output is constructed, new lines are included in the colored string.

### CODE REVIEW

**Comment by eladchen** [comment]
Whoa ! The unit tests are nearly impossible to make out.

There's two reasons why the build is failing:
1. The code was committed without using the npm task 'beautify'.
2. The tests are failing due to the output structure being changed.

I can fix the first issue, but the second is nearly impossible to make out. Those tests 'results.txt' are not human friendly.

**Comment by royriojas** [comment]
@eladchen. I can help with the other tests. fix the beautify and push, I can help with the tests. They are more like snapshot testing than actually unit testing (I mean I was not expecting a human generating those outputs :) )

In any case as I said I can fix those once beautification is fixed.

File: index.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ var getKeyLink = function(key) {
 };
 
 var printSummary = function(hash, title, method) {
-  var res = '\n\n' + chalk[method](title + ':\n');
+  var res = '\n\n' + chalk[method](title + ':') + chalk.white('\n');
```
Comment (royriojas) [inline index.js]: mmm, not sure it should be white, why not just `\n` instead? with no color control chars?

Comment (eladchen) [inline index.js]: Using plain '\n' doesn't preserve new lines when it is printed with gulp, I assume gulp is trimming the log before outputting it.

**Comment by royriojas** [comment]
Sorry man for the long delay on this branch, can you run `npm run lint` on this branch to make sure files are beautified? that is the only thing missing in this branch and it will be merged.

**Comment by eladchen** [comment]
I'll do it asap.

**Comment by royriojas** [comment]
I manually merged this PR. thanks @eladchen Sorry for the big delay in working on this.

**Pull request closed without merging**